taskcluster/ci/config.yml

```diff
@@ -0,0 +1,48 @@
+---
+trust-domain: scriptworker
+
+task-priority: high
+
+taskgraph:
+    register: scriptworker_taskgraph:register
+    repositories:
+        scriptworker:
+            name: "Scriptworker-scripts"
+            project-regex: scriptworker-scripts$
+            default-repository: https://github.com/mozilla-releng/scriptworker-scripts
+            default-ref: master
+            type: git
+
+workers:
+    aliases:
+        b-linux:
+            provisioner: scriptworker-{level}
```

Comment:
Workers are up and running looking at https://hg.mozilla.org/ci/ci-configuration/file/tip/worker-pools.yml#l2379 so this looks good 👍

Reply:
Thanks for the doublecheck!